client/webserver/webserver.go

```diff
@@ -845,6 +847,7 @@ func readPost(w http.ResponseWriter, r *http.Request, thing interface{}) bool {
 type userInfo struct {
 	Authed           bool
 	PasswordIsCached bool
+	IsInitialized    bool
```

Comment:
Are you populating this field for every single request even though it's only being used in one template? If so, please add the field to that template's page data instead.

Reply:
> please add the field to that template's page data instead.

yh.